src/Explorer.js

```diff
@@ -865,43 +1231,49 @@ class AbstractArgView extends React.PureComponent<AbstractArgViewProps, {}> {
 type AbstractViewProps = {|
   implementingType: GraphQLObjectType,
   selections: Selections,
-  modifySelections: (selections: Selections) => void,
+  modifySelections: (selections: Selections, commit: boolean) => ?DocumentNode,
```

Comment:
Instead of a boolean, can we have this take an optional object of type `{commit: boolean}` and have it default to true if you don't provide the map?

Reply:
Would also be good to force it to return something so that flow can help you if you accidentally forget to return. The type `DocumentNode | null` might do the trick, but I haven't tested it.